nemo/export/quantize/quantizer.py

```diff
@@ -167,80 +98,73 @@ def dummy():
         set_data_parallel_group(mpu.get_data_parallel_group())
         set_tensor_parallel_group(mpu.get_tensor_model_parallel_group())
 
+        model.set_inference_config(OmegaConf.to_container(inference_config))
+
     @staticmethod
-    def _load_and_modify_config(
-        model_file: str,
-        tensor_model_parallel_size: Optional[int] = None,
-        pipeline_model_parallel_size: Optional[int] = None,
-    ):
-        model_cfg = load_config(model_file)
-
-        with open_dict(model_cfg):
-            model_cfg.activations_checkpoint_method = None
-            model_cfg.activations_checkpoint_granularity = None
-            model_cfg.sequence_parallel = False
-            if tensor_model_parallel_size is not None:
-                model_cfg.tensor_model_parallel_size = tensor_model_parallel_size
-            if pipeline_model_parallel_size is not None:
-                model_cfg.pipeline_model_parallel_size = pipeline_model_parallel_size
```

Comment:
this part is automatically taken care of since it is moved to `model:` section in the yaml

Reply:
Do you mean model section from model_config.yaml or megatron_quantization.yaml modified here? Or please just link the piece of code that updates parallelism settings based on appropriate model section.

It's important to easily specify parallelism settings as this typically is going to be different than settings used in model_config.yaml for example, during training

Reply:
When model is loaded, it read `model:` section from `model_config.yaml` but then we overwrite values from `model:` section in `megatron_quantization.yaml` after loading the model so we can have the final configuration we want to use for PTQ/QAT.

If we look at training based example in https://github.com/NVIDIA/NeMo/blob/main/.github/workflows/cicd-main.yml#L4258 we can see how they define TP/PP, Activation checkpointing, model restore path etc under `model.` section.

For QAT since it uses the training pipeline, we need to use the parameters how they do it in training yaml files. The question is whether we want to have similar format for PTQ or have it different than QAT